workflow_manager/docker-entrypoint.sh

```diff
@@ -87,7 +87,7 @@ echo 'PostgreSQL is available'
 # Wait for Redis service.
 echo 'Waiting for Redis to become available ...'
 # From https://stackoverflow.com/a/39214806
-until [ +PONG = "$( (exec 8<>/dev/tcp/redis/6379 && echo -e 'PING\r\n' >&8 && head -c 5 <&8; exec 8>&-) 2>/dev/null )" ]; do
+until [ +PONG = "$( (exec 8<>/dev/tcp/$REDIS_HOST/6379 && echo -e 'PING\r\n' >&8 && head -c 5 <&8; exec 8>&-) 2>/dev/null )" ]; do
```

Comment:
Please change `$REDIS_HOST` to `${REDIS_HOST:-redis}`. That will ensure that when `$REDIS_HOST` is not set, `redis` is used as the host name.

Reply:
Great point. Done